src/utils/processOutputInterpreters/kim-build-output.js

```diff
@@ -0,0 +1,18 @@
+const LineSplitter = /\r?\n/;
+
+export default class KimBuildOutputCuller {
```

Comment:
Not sure why we need a culler that… does nothing? It's not like we have interfaces here (none of this is TypeScript) that we need to satisfy…

Reply:
Because of the nature of the build output. It arrives in numbered segments, with a `#n ` at the start of each line in segment `n`. We might want to show only the most recent segment, and remove the other parts. There's a lot of repetition in the early segments. Then you get to the part that processes the Dockerfile, and there's really no culling left to do.

So I left that in for the flexibility of doing something later (I know, YAGNI...).